spec/models/export/activity_commitment_column_spec.rb

```diff
@@ -0,0 +1,52 @@
+RSpec.describe Export::ActivityCommitmentColumn do
+  describe "#headers" do
+    it "returns an array that contains the header name of the column" do
+      activities = []
+
+      column = described_class.new(activities: activities)
+
+      expect(column.headers).to eql ["Original Commitment"]
+    end
+  end
+
+  describe "#rows" do
+    context "when there are no acitvities" do
+      it "returns an empty hash" do
+        activities = []
+
+        column = described_class.new(activities: activities)
+
+        expect(column.rows).to be {}
+      end
+    end
+
+    context "when there are acitivites" do
+      context "when the activity has a commitment" do
+        it "returns the activity ID and the value of the commitment" do
+          commitment = double(Commitment, value: 100000.00)
+          acitivity = double(Activity, commitment: commitment, id: "ACTIVITY_ID")
+          activities = [acitivity]
+
+          column = described_class.new(activities: activities)
+
+          expect(column.rows.count).to eql 1
+          expect(column.rows.first[0]).to eql "ACTIVITY_ID"
+          expect(column.rows.first[1]).to eql 100000.00
+        end
+      end
+
+      context "when the activity has no commitment" do
+        it "returns the value as 0" do
```

Comment:
Should the spec description say "returns the value as an empty string", or am I missing something?

Reply:
This is 'returns nil' now 👍